biojava-structure/src/main/java/org/biojava/nbio/structure/asa/AsaCalculator.java

```diff
@@ -317,36 +361,118 @@ public AsaCalculator(Point3d[] atomCoords, double probe, int nSpherePoints, int
 	}
 
 	/**
-	 * Returns list of indices of atoms within probe distance to atom k.
-	 * @param k index of atom for which we want neighbor indices
+	 * Returns the 2-dimensional array with neighbor indices for every atom.
+	 * @return 2-dimensional array of size: n_atoms x n_neighbors_per_atom
```

Comment:
adjacency list vs adjacency matrix...possible prefer one or the other universally based on expected density of graph